examples/src/RecordMovieComponent.tsx

```diff
@@ -0,0 +1,62 @@
+import React, { useState, useEffect, useRef } from "react";
+
+interface RecordMovieComponentProps {
+    isRecording: boolean;
+    setIsRecording: (isRecording: boolean) => void;
+}
+
+const RecordMovieComponent = ({
+    isRecording,
+    setIsRecording,
+}: RecordMovieComponentProps) => {
+    let [recordingDuration, setRecordingDuration] = useState<number>(0);
+    let [recordingStatus, setRecordingStatus] =
+        useState<string>("Not recording");
```

Comment:
Does this state need to exist? It seems like you could remove this, and just have the text switch between "Not recording" and "Recording..." based on the `isRecording` prop.

Reply:
You're right this is a stow away from the previous implementation.